heron/scheduler-core/src/java/com/twitter/heron/scheduler/UpdateTopologyManager.java

```diff
@@ -104,7 +139,6 @@ public void updateTopology(final PackingPlans.PackingPlan existingProtoPackingPl
     assertTrue(newContainerCount + removableContainerCount == 0 || scalableScheduler.isPresent(),
```

Comment:
Hi, @billonahill . I'm reading this part of code. I was wondering if we could use `Preconditions.checkArgument` or `Preconditions.checkState` provided by Google guava instead of assertTrue method?